### ISSUE TITLE
Embedding the playground

### ISSUE THREAD

**Issue by rdm** [jsoftware/j-playground#101]
It seems like the playground should be embeddable in a page.

Something like tinymce, but instead of a "what you see is what you get" editor, a J sentence which if clicked on becomes a text area which is a J session.

This could be used in J documentation on the wiki - examples for new users could become live examples if the user were sufficiently interested.

But, currently, the j playground is not packaged in a way which supports this.

Getting everything right in a first version might be counter-productive - for a first version, it would be better to show how this might look in "kit form" (thus allowing potential integration with other page design mechanisms), or "sketch form" (hinting at the possibilities).

But this seems like it would be quite useful, imperfections and all.

**Comment by cdburke** [comment]
Nice idea. Something like the playground Term window could pop up. I'll take a look.

**Comment by joebo** [comment]
I agree - nice idea. I have spent some time on https://www.uiua.org/ and
think their embedded approach is a good role model as well.

**Comment by rdm** [comment]
The uiua approach is interesting.

I was thinking of something slightly different. (Basically, textarea "console" which expands to fit content (this.style.height = this.scrollHeight + "px"') without any chrome, and an "are you sure" message when leaving the page if one of these consoles has been opened on the page.)

But the potential for different approaches suggests at least an initial version which is more a kit than a shrink wrapped module.
